grafana/src/eaOverview.jsonnet

```diff
@@ -228,37 +228,37 @@ local cacheEntryGetsPerSecond = addSideLegend(graphPanel.new(
   datasource=cortexDataSource,
 ).addTarget(
   prometheus.target(
-    'sum(rate(cache_data_get_count{' + namespaceFilter + '}' + interval + ')) by (service)',
+    'sum(rate(cache_data_get_count{' + masterFilter + '}' + interval + ')) by (service)',
     legendFormat='{{service}}',
   )
 ));
 
 
-local cacheValues = addSideLegend(graphPanel.new(
-  title='$feed Cache values',
-  format='none',
-  datasource=cortexDataSource,
-  repeat='feed',
-).addSeriesOverride(
-  {
-    alias: '/.*Median.*/',
-    color: 'rgb(255, 255, 255)',
-    fill: 0,
-    linewidth: 2,
-    zindex: 3,
-  }
-).addTargets(
-  [
-    prometheus.target(
-      'quantile(0.5, cache_data_get_values{feed_id=~"$feed.*",namespace="$namespace"}) by (feed_id)',
-      legendFormat='Median',
-    ),
-    prometheus.target(
-      'cache_data_get_values{feed_id=~"$feed.*",' + namespaceFilter + '}',
-      legendFormat='{{service}}',
-    ),
-  ]
-));
+// local cacheValues = addSideLegend(graphPanel.new(
```

Comment:
I'd remove this instead of commenting it out, and isolate it to one commit that can be reverted to later. Another option is to make this panel optional, hidden behind a feature flag.